src/assets/simp-adapter/build/simp-adapter.spec

```diff
@@ -36,8 +36,6 @@ Summary: SIMP Adapter for the Puppet Enterprise Puppet Installation
 License: Apache-2.0
 Requires: rsync
 Requires(post): puppet-agent
-Requires(post): pe-puppetserver
```

Comment:
This must be put back.  This is what differentiates between the simp-adapter-foss and simp-adapter-pe packages built by this spec file.